elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -475,9 +475,11 @@ export class PeerPool extends EventEmitter {
 			peerLimit,
 		});
 
-		[...peersToConnect, ...disconnectedFixedPeers].forEach(
-			(peerInfo: P2PPeerInfo) =>
-				this._addOutboundPeer(peerInfo, this._nodeInfo as P2PNodeInfo),
+		[
```

Comment:
I prefer the previous version but if the chosen prettier rules are set to this I'm ok with it.